pkg/controller/node/nodecontroller.go

```diff
@@ -368,6 +371,29 @@ func (nc *NodeController) maybeDeleteTerminatingPod(obj interface{}) {
 	}
 }
 
+// cleanupOrphanedPods deletes pods that are bound to nodes that don't
+// exist.
+func (nc *NodeController) cleanupOrphanedPods() {
+	pods, err := nc.podStore.List(labels.Everything())
+	if err != nil {
+		utilruntime.HandleError(err)
+		return
+	}
+
+	for _, pod := range pods {
+		if pod.Spec.NodeName == "" {
+			continue
+		}
+		if _, exists, _ := nc.nodeStore.Store.GetByKey(pod.Spec.NodeName); exists {
+			continue
+		}
+		fmt.Println("!!!!!!!!!!!!!!!!!!!!!!!!!!! %#v", nc.nodeStore.Store.List())
```

Comment:
I assume this was for debugging and should be removed or converted to a log message?